src/main/java/mate/academy/onlinebookstore/validation/IsbnValidator.java

```diff
@@ -7,12 +7,7 @@
 public class IsbnValidator implements ConstraintValidator<Isbn, String> {
 
     private static final String PATTERN_OF_ISBN = """
-             ^(?:ISBN
-             (?:-13)?:? )?(?=[0-9]{13}$
-             |(?=(?:[0-9]+[- ]){4})
-             [- 0-9]{17}$)97[89][- ]
-             ?[0-9]{1,5}[- ]?[0-9]+[- ]
-             ?[0-9]+[- ]?[0-9]$""";
+            \\d{9}[\\d|X]""";
```

Comment:
there is present @ISBN annotation implemented by jakarta.validation already. 

use it to reduce code amount and simplify your code to understand